web/src/components/MemoEditor.tsx

```diff
@@ -76,6 +76,13 @@ const MemoEditor: React.FC = () => {
     prevGlobalStateRef.current = editorState;
   }, [state, editorState.editMemoId]);
 
+  const handleInsertMark = (mark: string) => {
+    if (!editorRef.current || mark) {
+      return;
+    }
+    editorRef.current.insertText(mark, true);
```

Comment:
```suggestion
    editorRef.current?.insertText(mark, true);
```

I don't know why currently the shortcut keys are invalid. After this modification, it works